src/app/hooks/useWalletReducer.ts

```diff
@@ -151,6 +153,8 @@ const useWalletReducer = () => {
       stxPublicKey: wallet.stxPublicKey,
       bnsName: wallet.bnsName,
     };
+    trackMixPanel('Restore wallet');
+
```

Comment:
we should start a better semantic naming system for event keys, and keep them consistent with mobile event keys.

Reply:
Good idea, I added the enum for this purpose in this xverse-core PR: https://github.com/secretkeylabs/xverse-core/pull/240